packages/button/stories/index.js

```diff
@@ -142,25 +133,34 @@ loadingExample.add('lone icon', _ => (
     loading
   />
 ))
+
 class SwitchToLoading extends React.Component {
-  constructor(props) {
+  constructor (props) {
     super(props)
     this.state = { loading: false }
   }
-  componentDidMount() {
+
+  componentDidMount () {
     this.timeout = setTimeout(_ => {
       this.setState({ loading: !this.state.loading })
     }, 500)
   }
-  componentWillUnmount() {
+
+  componentWillUnmount () {
     clearInterval(this.timeout)
   }
-  render() {
+
+  render () {
     return React.cloneElement(this.props.children, {
       loading: this.state.loading
     })
   }
 }
+
+SwitchToLoading.propTypes = {
```

Comment:
Is linting bringing this to your attention, or are you just that sharp?! :) 

What's your workflow like?  Do you have a lint plugin for vscode or are you running npm run lint now and again?